src/components/tabs/tabs.hbs

```diff
@@ -11,6 +11,9 @@
         <a id="{{linkId}}" class="bx--tabs__nav-link" href="javascript:void(0)" role="tab" aria-controls="{{panelId}}"{{#if selected}} aria-selected="true"{{/if}}>{{label}}</a>
       </li>
     {{/each}}
+      <li class="bx--tabs__nav-item" data-target=".tab-6" role="presentation" disabled>
```

Comment:
Wondering if we can merge here to the `{{#each}}` loop? We may add `{{#if disabled}} disabled{{/if}}` and update the `tabs.config.js` accordingly.

Reply:
@asudoh great suggestion! I went ahead and did that. 😄